run-agent.sh

```diff
@@ -66,7 +66,7 @@ else
    prepare_conf
 fi
 
-${AGENT_DIST}/bin/agent.sh start
+gosu buildagent ${AGENT_DIST}/bin/agent.sh start
```

Comment:
Consider chown for `buildagent` user during 'initialization' (in addition to the Dockerfile),  either:
A) in this entrypoint 
B) As an entry in /services/

```
    chown -R buildagent:buildagent /opt/buildagent
```

Otherwise this may break for users of `jetbrains/teamcity-agent` which run "Docker from the host".  The permissions need to be set at container 'runtime' since the checkout dir and temp dir are bind-mount VOLUMEs; as explained here:    https://hub.docker.com/r/jetbrains/teamcity-agent/

>Options -v /opt/buildagent/work:/opt/buildagent/work and -v /opt/buildagent/temp:/opt/buildagent/temp are required if you want to use Docker Wrapper on this build agent.

Reply:
@stayclassychicago done